source/AI.cpp

```diff
@@ -2449,11 +2453,18 @@ bool AI::MoveTo(Ship &ship, Command &command, const Point &targetPosition,
 	double maxVelocity = ship.MaxVelocity(ShouldUseAfterburner(ship)) * .99;
 	if(isFacing && (velocity.LengthSquared() <= maxVelocity * maxVelocity
 			|| dp.Unit().Dot(velocity.Unit()) < .95))
+	{
 		command.SetThrust(1.);
+		double deviation = ship.Velocity().Unit().Cross(ship.Facing().Unit());
+		command.SetLateralThrust(deviation * 5);
+	}
+
 	else if(shouldReverse)
 	{
 		command.SetTurn(TurnToward(ship, velocity));
 		command.SetThrust(-1.);
+		double deviation = ship.Velocity().Unit().Cross(ship.Facing().Unit());
+		command.SetLateralThrust(deviation * 5);
```

Comment:
I'm not sure about this section.

Does it need stabilization at all? 

Or more radically, if it is actually using reverse thrust to accomplish whatever the AI is trying to do, should it be using the *opposite* lateral thrust? I don't think so, but it's worth thinking about.